azavea/pfb-network-connectivity
# Save census block count to AnalysisJob

## Overview

Adds a 'set_job_attr' management command and uses it to save the census
block count onto a job during import.

### Demo

![image](https://cloud.githubusercontent.com/assets/6598836/24359935/cf6fe1d8-12d3-11e7-994d-02fe9d361549.png)

### Notes

It prints a warning but doesn't raise an except for missing or invalid job, which is preferable in development where jobs run directly don't have valid job IDs and might make sense in production as well.  Other failures result in exceptions.


## Testing Instructions

Check out, run migrations, and run a job.  The census block count attribute should be populated as soon as that part of the import process is done.

Resolves #183.
